app/tests/spec/views/settings.js

```diff
@@ -294,7 +506,7 @@ function (chai, $, sinon, View, RouterMock, TestHelpers,
           });
           sinon.spy(user, 'clearSignedInAccount');
 
-          return view.submit()
+          return view.signOut()
```

Comment:
I really like this change, I always felt like it was a bad decision on my part to use the submit handler to sign the user out.